src/components/cars/DeleteCar.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { deleteCar, fetchCars } from '../../redux/features/carsAction';
+import Navbar from '../Navbar';
+
+function DeleteCar() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const DeleteCar= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪